sklearn/decomposition/factor_analysis.py

```diff
@@ -347,3 +359,38 @@ def score(self, X, y=None):
             Average log-likelihood of the samples under the current model
         """
         return np.mean(self.score_samples(X))
+
+    def _rotate(self, method="varimax", n_components=None, tol=1e-6):
+        "Rotate the factor analysis solution."
+        implemented = {"varimax", "quartimax"}
+        if method in implemented:
+            self.components_ = self._ortho_rotation(
+                self.components_.T, method=method, eps=tol)[:n_components]
+        else:
+            raise NotImplementedError("'method' must be in %s, not %s"
```

Comment:
Usually we would use a `ValueError`. Subtle, yes, but `NotImplementedError` is usually for a function call with argument types where the operation is not yet supported, for instance.